web/src/admin/users/UserImpersonateForm.ts

```diff
@@ -0,0 +1,39 @@
+import { DEFAULT_CONFIG } from "@goauthentik/common/api/config";
+import "@goauthentik/components/ak-text-input";
+import { Form } from "@goauthentik/elements/forms/Form";
+
+import { msg } from "@lit/localize";
+import { TemplateResult, html } from "lit";
+import { customElement, property } from "lit/decorators.js";
+
+import { CoreApi, ImpersonationRequest } from "@goauthentik/api";
+
+@customElement("ak-user-impersonate-form")
+export class UserImpersonateForm extends Form<ImpersonationRequest> {
+    @property({ type: Number })
+    instancePk?: number;
+
+    async send(data: ImpersonationRequest): Promise<void> {
+        return new CoreApi(DEFAULT_CONFIG).coreUsersImpersonateCreate({
+            id: this.instancePk || 0,
+            impersonationRequest: data,
+        });
+        // .then(() => {
+        //     window.location.href = "/";
+        // });
```

Comment:
Assuming this is commented out for testing?

Reply:
yeah, this shouldn't be commented out

Reply:
let me just try out again if that works

Reply:
yep, all good